tox.ini

```diff
@@ -58,6 +58,7 @@ whitelist_externals = make
 commands =
     pip install pyramid[docs]
     make -C docs html epub BUILDDIR={envdir} "SPHINXOPTS=-W -E"
```

Comment:
Just update this to include doctests, since you already added a new doctest target in the Makefile.

    make -C docs doctest html epub BUILDDIR={envdir} "SPHINXOPTS=-W -E"